src/LazyLoadImage/LazyLoadImage.tsx

```diff
@@ -70,9 +72,10 @@ export default function LazyLoadImage({
         <Skeleton
           sx={{
             borderRadius: 0,
-            height: dimensions.height,
+            display: "block",
+            height: autoFitSkeleton ? "100%" : dimensions.height,
             transform: "none",
```

Comment:
add a `marginTop: 0.8` to avoid the jumb of when actual image appears.

Reply:
I also applied the fix you suggested on that PR to the storybook here